js_modules/dagster-ui/packages/ui-core/src/graphql/schema.graphql

```diff
@@ -2906,6 +2906,7 @@ enum SensorType {
 type AssetSelection {
   assetSelectionString: String
   assetKeys: [AssetKey!]!
+  assets: [Asset!]!
```

Comment:
Just a small call-out, but I think we have both an `Asset` GQL type and an `AssetNode` (SDA) type, and it would probably make more sense to use AssetNode here. Currently we're accessing the `asset.definition` to get the AssetNode and I think these are always SDAs?